src/graphics/bargraphics.jl

```diff
@@ -30,8 +32,8 @@ function BarplotGraphics(
         char_width::Int,
         transform = identity;
         color::UserColorType = :green,
-        symb = "■") where {R <: Number}
-    BarplotGraphics(bars, char_width, crayon_256_color(color), symb, transform)
+        symbols = ["■"]) where {R <: Number}
+    BarplotGraphics(bars, char_width, crayon_256_color(color), symbols, transform)
```

Comment:
Changing keyword names can be breaking so I think we should deprecate it for every place...

```suggestion
        symb = nothing, # deprecated
        symbols = ["■"]) where {R <: Number}
    if !isnothing(symb)
        depwarn("keyword \"symb\" is deprecated in favor of \"symbols\"", :BarplotGraphics)
        symbols = [symb]
    end
    BarplotGraphics(bars, char_width, crayon_256_color(color), symbols, transform)
```